docs/examples/staticfiles.rst

```diff
@@ -13,10 +13,25 @@ So ``http://localhost:8080/static/img.gif`` should return an image and ``http://
 
     @route('/static/', branch=True)
     def static(request):
-        return File("./static")
+        return File('./static')
 
     @route('/')
     def home(request):
         return '<img src="/static/img.gif">'
 
-    run("localhost", 8080)
\ No newline at end of file
+    run('localhost', 8080)
+
+To setup a route to a single :api:`twisted.web.static.File <t.w.static.File>` pass ``branch=True`` to the route and set ``isLeaf=True`` on the file resource.
```

Comment:
Why is `isLeaf=True` necessary?  If it's a *single* file, why is `branch=True` necessary?  I apologize if we've discussed this already, but I couldn't find it in the history, and I'm a bit confused.